<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>jovo</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">
    <!-- font awesome -->
    <link rel="stylesheet" href="font-awesome/css/font-awesome.css">

    <!-- Custom CSS -->
    <link href="css/main.css" rel="stylesheet">

    <!-- ocp icon -->
    <link rel="shortcut icon" href="ocp_main3.ico">

    <!-- bower:css -->
    <!-- <link rel="stylesheet" href="/content/themes/jhu_id/bower_components/normalize-css/normalize.css"> -->
    <link rel="stylesheet" href="fonts/gentona/gentona.css">
    <link rel="stylesheet" href="fonts/titling-gothic/titling-gothic.css">
    <link rel="stylesheet" href="fonts/quadon/quadon.css">
    <link rel="stylesheet" href="fonts/arnhem/arnhem.css">
    <!-- endbower -->

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
        <![endif]-->

    </head>

    <!-- The #page-top ID is part of the scrolling feature - the data-spy and data-target are part of the built-in Bootstrap scrollspy function -->

    <body id="page-top" data-spy="scroll" data-target=".navbar-fixed-top">

        <!-- Navigation -->
        <nav class="navbar navbar-default navbar-fixed-top top-nav-collapse" role="navigation">
            <div class="container">
                <div class="navbar-header page-scroll">
                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
                        <span class="sr-only">Toggle navigation</span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                        <span class="icon-bar"></span>
                    </button>
                    <a class="navbar-brand page-scroll" href="#page-top">jovo.me</a>
                </div>

                <!-- Collect the nav links, forms, and other content for toggling -->
                <div class="collapse navbar-collapse navbar-ex1-collapse">
                    <ul class="nav navbar-nav">
                        <!-- Hidden li included to remove active class from philosophy link when scrolled up past philosophy section -->
                        <li class="hidden">
                            <a class="page-scroll" href="#page-top"></a></li>
                        <li><a class="page-scroll" href="#philosophy">philosophy</a></li>
                        <li><a class="page-scroll" href="#research">research</a></li>
                        <li><a class="page-scroll" href="#code">code</a></li>
                        <li><a class="page-scroll" href="#links">links</a></li>
                        <li><a class="page-scroll" href="#teaching">teaching</a></li>
                        <!-- <li><a class="page-scroll" href="#funding">funding</a></li> -->
                        <li><a class="page-scroll" href="#contact">contact</a></li>
                        <li><a class="page-scroll" href="#hiring">hiring</a></li>
                        <li><a class="fa fa-external-link" href="https://www.dropbox.com/s/xg96unmbvjurwa2/jovo_cv.pdf?dl=0" target="_blank">cv</a></li>


                    </ul>
                </div>

            </div>
            <!-- /.container -->
        </nav>



        <!-- about me  -->
        <section id="about me" class="first-section">
            <div class="container">
                <div class="item row">
                    <div class="col-md-2 col-sm-10 col-xs-12">
                        <h1> about me </h1>
                        <img class="img-responsive" src="jovo.png"/>
                    </div>

                    <div class="desc col-md-10 col-sm-10 col-xs-12">
                     <h1 class="lineup-hack" style="opacity: 0">me</h1>
                     <!-- ^ you better look at this comment jovo because this css hack makes it line up -->
                     <p> I am an Assistant Professor in the Department of <a href="http://www.bme.jhu.edu/">Biomedical Engineering</a> and the <a href="http://www.icm.jhu.edu/">Institute for Computational Medicine</a> at <a href="https://www.jhu.edu/">Johns Hopkins University</a>.  I sit at the <a href="http://cis.jhu.edu/">Center for Imaging Science</a> in Clark Hall at the Homewood campus.  I also have an appointment in the <a href="http://idies.jhu.edu/">Institute for Data Intensive Engineering and Sciences</a>.  My work focuses largely on big and wide data, especially neuroscience, focusing on statistics of brain graphs (connectomes).  I co-founded the <a href="http://ocp.me">Open Connectome Project</a> (OCP) with my brother R. Jacob Vogelstein and <a href="http://hssl.cs.jhu.edu/~randal/">Randal Burns</a>, Associated Professor in the Department of Computer Science at Johns Hopkins University. Recently it has spawned <a href="http://neurodata.io">NeuroData</a> which is the mother of several complementary projects, including  OCP, Open Synaptome Project, and more.
                        We run a very vertical group, with people working at all levels of analysis, ranging from data collection to analysis and interpretation.  We are always looking for new collaborators and team members.  Please <a href="mailto:jovo@jhu.edu">inquire</a> if you are interested. 
                    </p>
                </div>
            </div>
        </div>
    </section>



    <!-- philosophy Section -->
    <section id="philosophy" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>philosophy</h1>
                    <p>We seek patterns in our physical worlds (e.g., our bodies, our brains) as well as our mental worlds (e.g., our perceptions, experiences, memories, thoughts, emotions, psychiatric conditions).  More importantly, we seek to understand patterns in our mental worlds <em>in terms of</em> our physical worlds. Our hope and belief is that via developing a deeper understanding of the links between these worlds, we will be able to bring them into greater alignment. A primary motivating factor is that all humans/animals have brains and therefore, such ideas could directly benefit all of animalkind. Thus, all of our research products are freely available to all.  For more details see my 
                    <a href="https://www.dropbox.com/s/xg96unmbvjurwa2/jovo_cv.pdf?dl=0" target="_blank">cv</a>.
                    </p>
                </div>
            </div>
        </div>
    </section>

    <!-- research Section -->
    <section id="research" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>research</h1>
                    <p>The fundamental driving force of science is the discovery of latent structure that converts myriad disparate data  into understanding. In the 21st century, the growth of data acquisition has greatly outpaced the growth of data analysis, rendering current computational and statistical tools insufficient for extracting meaning from large datasets. Neuroscience is particularly susceptible to these big data challenges, as neuroexperimentalists have devised methods of collecting terabytes of data per hour. Without statistical and computational frameworks for analysis and organization of these data, the field will be unable to fully reap the benefits of this enormous potential. Our expertise in (i)  computer science, (ii) data curation, (iii) statistical science,  and (iv) neuroscience  enables us to fill this gap. Below we elaborate on these four threads.
                    </p>

                    <h3>statistical science</h3>

                    <p>Statistical sciences have developed a beautiful corpus of knowledge on inference in various settings over the last 100 years.  However, in recent years, new experimental and data collection methods have yielded datasets that no longer fit within the assumptions upon which that body of literature rests.  We therefore  focus  on building statistical methods for 21st century data, including:
                        <ul>
                            <li><strong>time-series</strong> 
                                <a href="https://github.com/jovo/smc-oopsi">sequential monte carlo</a>, <a href="https://github.com/jovo/oopsi">non-negative deconovlution</a>, <a href="https://github.com/jovo/spike-sorting">dictionary learning</a>, <a href="https://github.com/decarlson/opass">variational inference</a>, and <a href="https://github.com/shachen/PLDS"> high-dimensional system identification</a>;</li>
                                <li><strong>graphs</strong> inference of graphs, supervised and unsupervised vertex clustering, similarity and dissimilarity metrics, graph matching, and population theory including graph classification and joint embedding;</li>

                                <li><strong>shapes</strong> <a href="https://github.com/mkazhdan/ShapeSPH">non-parametric shape descriptors</a> and  <a href="http://cis.jhu.edu/software/">nonlinear diffeomorphisms</a> for statistical analysis of big shapes;</li>


                                <li><strong>vectors</strong> high-dimensional <a href="https://github.com/jovo/LOL">linear</a> and <a href="https://github.com/ttomita/RandomerForest">non-linear classification</a>, optimal design, signal detection, <a href="https://nips.cc/Conferences/2013/Schedule?q=vogelstein">density estimation</a>,  and dependence discovery.</li>
                            </ul>
                        </p>

                    <h3>computer science</h3>

                    <p>Statistical methods require computational implementations. Moreover, the raw data of the 21st century is not typically in a form immediately amenable to statistical analysis; rather, various data wrangling/munging/pre-processing is typically beneficial.   We therefore develop computational tools to enable principled statistical inference on such data.  This includes:

                        <ul>
                        <li><strong>petascale data management</strong> including <a href="https://github.com/neurodata/open-connectome"><em>spatial databases</em></a> for 3D+ (multispectral or time-series) image data including voxel-wise metadata, and  <a href="https://github.com/neurodata/m2g"><em>grute databases</em></a>  for populations of massive graphs with rich attributes, as well as MATLAB and Python APIs, a content distribution system, LIMS, and hardware configuration;</li> 

                            <li><strong>interactive visualizations</strong> including for multispectral annotated 3D+ big <a href="https://github.com/neurodata/NeuroDataViz">images</a>, <a href="http://gx.neurodata.io">graphs</a>, and <a href="http://vx.neurodata.io">vectors</a>.</li>

                            <li><strong>streaming image processing</strong> including both <a href="https://github.com/mkazhdan/DMG">2D stitching artifact correction</a>  and <a href="http://www.cs.jhu.edu/~misha/Code/GradientDomainFusion/Version1.0/">3D</a>  histogram normalization;</li>

                            <li><strong>distributed computer vision</strong> including graph inference from <a href="http://i2g.io/">serial electron microscopy</a>, calcium imaging,  <a href="http://fcp-indi.github.io/">functional magnetic resonance imaging</a>, <a href="https://github.com/neurodata/ndmg">diffusion tensor imaging</a> , and object detection in both <a href="http://docs.neurodata.io/vesicle/">MATLAB</a> and <a href="http://docs.neurodata.io/nddocs/#parse">Python</a>;</li>

                            <li><strong>scalable machine learning libraries</strong> (<a href="http://flashx.io">FlashX</a>) including FlashGraph for billion vertex graph analytics using various graph traversal algorithms, FlashMatrix for matrix applications such as svd, pca, k-means, and linear classification, and FlashR for usability;  </li>



                        </ul>
                    </p>

                    <h3 id="data">big data curation</h3>

                    <p>To answer any of the below questions, using any of the above statistical methods and computational tools, fundamentally requires data.  Much like all of our code is open source, all of our data is open access.  Moreover, we provide it in a fashion immediately amenable to analysis for everyone, including the below data types:

                        <ul>
                            <li><strong>images</strong> both anatomical including electron microscopy, array tomography, expansion microscopy, x-ray microscopy, CLARITY, histology, as well as  physiological images such as calcium imaging, massively parallel electrophysiology, and multi-modal mri;</li>

                            <li><strong>annotations</strong> including volumetric annotations, skeletonizations, and ROIs;</li>

                            <li><strong><u>g</u>raphs</strong> with <u>r</u>ich attrib<u>ute</u>s ("<em>grutes</em>") derived from a wide variety of experimental techniques and scales.</li>

                        </ul>
                    </p>                    


                        <h3>neuroscience</h3>

                        <p>The above described tools, both statistical and computational, are designed and built in the service of answering fundamental questions in neuroscience, both basic and clinical.  Below are the projects we are have worked on, and continue to develop:

                          <ul>
                              <li><strong>anatomy</strong> including em connectomics, synaptomics, and mesoscale neuro-architecture; </li>

                              <li><strong>physiology</strong> including analysis of correlations from o-phys data;</li>

                              <li><strong>systems</strong> including <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0095753">biomarker discovery</a> from multimodal mri data, and <a href="http://classic.sciencemag.org/content/344/6182/386.abstract">behavioromics</a>;</li>

                              <li><strong>connectome coding</strong> is homologous to neural coding, where we learn the rules by which memories & experiences are stored in patterns of connectivity, rather than patterns of activity.</li>
                          </ul>
                      </p>



                      <p>For more information about any of these projects, or to tell me about your awesome new project, <a href="mailto:jovo@jhu.edu">email me</a>, or swing by <a class="page-scroll" href="#contact">my office</a> sometime!</p>
                  </div>
              </div>
          </div>
      </section>

      <section id="code" class="generic-section">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h1>code</h1>
                    <p>Essentially every project I embark on includes writing some code, if only to buttress the theoretical results with numerical examples.  Moreover, as an adherent to the philosophy of open science, code I write is always open source by the time of submission, and often sooner.  Thus,  <a href="https://github.com/jovo" target="_blank">my personal github account</a> contains many repos including code from publications prior to launching the   
                        <a href="https://github.com/openconnectome/" target="_blank">Open Connectome Project</a>. We have subsequently migrated everything we are still using to our new organization, <a href="https://github.com/neurodata/" target="_blank">neurodata</a>.  Finally, we also created and continue to develop <a href="http://flashx.io/" target="_blank">FlashX</a>. 
                        <!-- Nevertheless, here I will try to post the codes from various projects that I think are potentially useful for other applications.   -->
                    </p>

<!--                         <h3 id="research-topics"><strong>time series analysis</strong></h3> 
                        <ol>
                            <li>Fast Nonnegative Deconvolution for Spike Train Inference from Population Calcium Imaging. <a href="https://github.com/jovo/fast-oopsi/raw/master/fast-oopsi.pdf" target="_blank">manuscript</a>, <a href="https://github.com/jovo/fast-oopsi/raw/master/code/functions/fast_oopsi.m" target="_blank">code</a>, and <a href="https://github.com/jovo/fast-oopsi" target="_blank">repo</a> .</li>
                            <li>Spike inference from calcium imaging using sequential Monte Carlo methods.  <a href="https://github.com/jovo/smc-oopsi/raw/master/manuscript/VogelsteinPaninski09.pdf" target="_blank">article</a>, <a href="https://github.com/jovo/oopsi" target="_blank">code</a>.</li>
                            <li> Real-Time Inference for a Gamma Process Model of Neural Spiking. <a href="http://papers.nips.cc/paper/5061-real-time-inference-for-a-gamma-process-model-of-neural-spiking" target="_blank">manuscript</a>, <a href="https://github.com/decarlson/opass" target="_blank">code</a>.</li>
                        </ol>
                        <h3 id="research-topics"><strong>graph analysis</strong></h3> 
                        <ol>
                            <li>Graph Classification using Signal Subgraphs: Applications in Statistical Connectomics. <a href="http://arxiv.org/abs/1108.1427" target="_blank">arxiv</a>, <a href="https://github.com/jovo/MatGraphStat/tree/master/SignalSubgraphClassifier" target="_blank">code</a>, <a href="https://github.com/jovo/signal-subgraph-classifier" target="_blank">repo</a>, <a href="http://www.computer.org/csdl/trans/tp/2013/07/ttp2013071539-abs.html" target="_blank">article</a>.</li>
                            <li>Fast Approximate Quadratic Programming for Large (Brain) Graph Matching. <a href="http://arxiv.org/abs/1112.5507" target="_blank">arxiv</a>, <a href="https://github.com/jovo/MatGraphStat" target="_blank">code</a>.</li>
                            <li>Seeded Graph Matching Via Joint Optimization of Fidelity and Commensurability. <a href="http://arxiv.org/abs/1401.3813" target="_blank">arxiv</a>, <a href="https://github.com/igraph/xdata-igraph/blob/develop/interfaces/R/igraph/R/sgm.R" target="_blank">code</a>.</li>
                            <li>Robust Multimodal Graph Matching: Sparse Coding Meets Graph Matching. <a href="http://papers.nips.cc/paper/4925-robust-multimodal-graph-matching-sparse-coding-meets-graph-matching">nips</a>, <a href="http://arxiv.org/abs/1311.6425" target="_blank">arxiv</a>, <a href="http://www.fing.edu.uy/~mfiori/code/code_graph_matching.zip">code</a>. </li>
                            <li>  DELTACON: Measuring Connectivity Differences in Large Networks. <a href="http://dx.doi.org/10.1137/1.9781611972832.18">article</a>. <a href="http://arxiv.org/abs/1304.4657" target="_blank">arxiv</a>, <a href="http://www.cs.cmu.edu/~dkoutra/CODE/deltacon.zip" target="_blank">code</a>.
                                <li>MR Connectome Automated Pipeline. <a href="http://arxiv.org/abs/1111.2660">arxiv</a>, <a href="http://nitrc.org/projects/mrcap" target="_blank">code</a>, <a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?tp=&arnumber=6173097&contentType=Journals+%26+Magazines&sortType%3Dasc_p_Sequence%26filter%3DAND%28p_IS_Number%3A6173090%29" target="_blank">abstract</a>, <a href="http://db.tt/BMEVlrpE" target="_blank">pdf</a>.</li>
                            </ol> -->
                        </div>
                    </div>
                </div>
            </section>

            <section id="links" class="generic-section">
                <div class="container">
                    <div class="row">
                        <div class="col-lg-12">
                            <h1>links</h1>

                            <ul>
                                <li>jovo: <a href="http://scholar.google.com/citations?hl=en&user=DWPfdT4AAAAJ&sortby=pubdate&view_op=list_works&gmla=AJsN-F7eFaJivagMCwgIbNf4cOwHKPj8aFcIANb8rku8bPmHR88ZxVs8dbsn68pjjibAZ9I4KrtOJDCVC411dnpjg51-NPcMQ6SrgdFh3NXdXyjq9dimPVg" target="_blank">google scholar</a>, <a href="http://www.ncbi.nlm.nih.gov/myncbi/browse/collection/40260830/?sort=date&direction=ascending" target="_blank">pubmed</a>, <a href="http://arxiv.org/find/all/1/au:+Vogelstein_J/0/1/0/all/0/1" target="_blank">arXiv</a>, <a href="https://github.com/jovo" target="_blank">github</a>, <a href="https://www.dropbox.com/s/xg96unmbvjurwa2/jovo_cv.pdf?dl=0" target="_blank">cv</a>.</li>

                                <li>open connectome project: <a href="http://www.openconnectomeproject.org/" target="_blank">website</a>, <a href="http://www.github.com/openconnectome" target="_blank">github</a>, </li>

                                <li>neurodata: <a href="http://neurodata.io/" target="_blank">website</a>, <a href="http://www.github.com/neurodata" target="_blank">github</a></li>

                            </ul>
                        </div>
                    </div>
                </div>
            </section>

            <section id="teaching" class="generic-section">
                <div class="container">
                    <div class="row">
                        <div class="col-lg-12">
                            <h1>teaching</h1>
                            <ul>

                                <li><a href="https://github.com/Upward-Spiral-Science/" target="_blank">The Art of Data Science (Spring 2016)</a></li>

                                <li>Introduction to Computational Medicine I (Fall 2015)</li>

                                <li><a href="https://github.com/Statistical-Connectomics-Sp15/intro" target="_blank">Statistical Connectomics (Spring 2015)</a></li>
                            </ul>
                        </div>
                    </div>
                </div>
            </section>

            <!-- Contact Section -->
            <section id="contact" class="generic-section">
                <div class="container">
                    <div class="row">
                        <div class="col-lg-12">
                            <h1>contact</h1>
                            <p>jovo [at symbol] jhu [dot] edu<br>
                                <br>
                                Joshua T. Vogelstein<br>
                                Center for Imaging Science<br>
                                Clark Hall, Rm 317C<br>
                                3400 N. Charles St<br>
                                Baltimore, MD 21218
                            </p>
                        </div>
                    </div>
                </div>
            </section>

            <!-- Hiring Section -->
            <section id="hiring" class="generic-section">
                <div class="container">
                    <div class="row">
                        <div class="col-lg-12">
                            <h1>hiring</h1>
                                <p>NeuroData is always hiring exceptional individuals at all levels! In fact, even if we are not an optimal choice for your future development as a scientist, we endeavor to assist/support your growth in various ways, including short/extended visits to work with us.  Unfortunately, because our time and funding are limited, we cannot work with everyone that we would love to; nonetheless, please don't hesitate to contact us in case we can provide some guidance :)   We are currently particularly looking for people with their own ideas that they are super passionate about, that we might be able to contribute to, as well any of the <a href="research">above</a> listed projects.</p>

                                <p>Please email your cv (and transcript if appropriate) to jovo at jhu dot edu.  See below for more details:</p>

                    <ul>
                        <li><strong>summer intern:</strong> these are really difficult for us to support for anybody not local to JHU.  I recommend trying as much as you can to get research experience at your local university. </li>
                        <li><strong>research associate:</strong> an undergraduate degree in a related field, eg, biomedical engineering, neuroscience, or computer science.</li>
                        <li><strong>current JHU undergrads:</strong> i advise many undergraduates for credit, please email me your interests and your transcript, so i can find a topic of mutual interest or suggest a different faculty member more suited for you.</li>
                        <li><strong>current JHU masters students:</strong>  please contact me as soon as you can, so we can get started right away!</li>
                        <li><strong>current JHU PhD students:</strong>  great! Email me your interests and your transcript, and we'll find a project for you, or we'll find a faculty better suited for you. </li>
                        <li><strong>potential JHU graduate students: </strong> we are always accepting new students. the best plan is to apply to the <a href="http://www.bme.jhu.edu/graduate/phd/prospective-students">BME PhD program</a>, applications are typically due in late november/early december. i can also advise students in other departments, including biostats, neuroscience, and computer science, so i recommend applying widely to increase the chances that we can work together.</li>
                        <li><strong>postdocs:</strong> we are especially always hiring postdocs! please email me your particular interest in our group and attach your cv.</li>
                    </ul>

                </div>
            </div>
        </div>
    </section>


            <!-- Funding Section -->
            <section id="funding" class="generic-section">
                <div class="container">
                    <div class="row">
                        <div class="col-lg-12">

                    <p>This work is graciously supported by the Defense Advanced Research Projects Agency (DARPA) SIMPLEX program through SPAWAR contract N66001-15-C-4041, DARPA GRAPHS N66001-14-1-4028, and NIH-TRA 1R01NS092474,  "Synaptomes of Mouse and Man".</p>


                </div>
            </div>
        </div>
    </section>

    <section class="copyright">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <p class="copyright">designed by <a href="https://github.com/jovo/jovo.github.com/graphs/contributors" target="_blank">us</a>.   &copy; <a href="http://jovo.me">jovo.me</a>  2016</p>                    
                </div>
            </div>
        </div>
    </section>

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Scrolling Nav JavaScript -->
    <script src="js/jquery.easing.min.js"></script>
    <script src="js/main.js"></script>

</body>

</html>
